SpatialGDKEditorToolbar/Source/Private/SchemaGenerator/Utils/ComponentIdGenerator.h

```diff
@@ -13,6 +13,11 @@ struct FComponentIdGenerator
 		return InitialId + (NumIds++);
 	}
 
+	int GetCurrentId()
```

Comment:
Understand the desire for this, but it could easily break (recursive function etc.). Can we factor it out?

Reply:
How can it break exactly in a recursive function?

Reply:
It's only valid until you call GetNextAvailableId(). If you have another function calling that, and you didn't anticipate, you'd be using the wrong id. I think it's just simpler to retrieve the value once, and use it locally.